packages/radix-ui-themes/src/components/select.props.tsx

```diff
@@ -18,7 +18,7 @@ const selectTriggerPropDefs = {
   variant: { type: 'enum', className: 'rt-variant', values: triggerVariants, default: 'surface' },
   ...colorPropDef,
   ...radiusPropDef,
-  placeholder: { type: 'string' },
+  placeholder: { type: 'ReactNode' },
```

Comment:
This currently only accepts 'ReactNode' as a string, it should be:

```suggestion
  placeholder: { type: ReactNode },
```

And don't forget to import `ReactNode` from `react`.

Reply:
Oh yes you are totally right, late night review is never a good idea.